src/App/index.js

```diff
@@ -34,7 +34,7 @@ export default (props) => (
           )}
         />
         <Match
-          pattern={`/instructors/:instructor_id`}
+          pattern={`/instructors/:instructorId`}
```

Comment:
I updated variables to be camel case when used in the app, snake case when dealing directly with endpoint responses.